### ISSUE TITLE
Calendar 7.7 allow change of displayed fields on hover

### ISSUE THREAD

**Issue by mikesolo** [salesagility/SuiteCRM#2347]
#### Issue

When you hover over a Task on the Calendar it shows Due Date & Time, Status, Priority & Related to

It version 7.6 and previous you could easily change what was displayed using custom/modules/Tasks/metadata/additionalDetails.php

Please can we have a upgrade safe way of modifying this again

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Calendar 7.7 allow change of displayed fields on hover](https://trello.com/c/ZkWPJudu/67-calendar-7-7-allow-change-of-displayed-fields-on-hover)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
